app/routes/thematic/geoencoder.py

```diff
@@ -0,0 +1,216 @@
+import re
```

Comment:
@dmannarino I know you said you didn't like thematic. I was thinking of other ideas, is this admin area specific endpoints, maybe we just put it under like something like "political"? E.g. `/political/geoencoder`. Then the future can include additional GADM endpoints, WDPA, concessions, etc. Not sure if we need to distinguish it from the rest of the API, or we can just have it all in under a header in the docs.

Reply:
But I do think `geoencoder` is a little vague if it only works for admin areas, geoencoding implies converting any text of the place into coordinates: https://en.wikipedia.org/wiki/Address_geocoding

Reply:
Changed to /political/id-lookup